mmv1/products/sql/terraform.yaml

```diff
@@ -62,6 +72,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       onPremisesConfiguration.port: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validation.IntBetween(0, 65535)'
+      onPremisesConfiguration.password: !ruby/object:Overrides::Terraform::PropertyOverride
+        sensitive: true
+      onPremisesConfiguration.client_key: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
```suggestion
      onPremisesConfiguration.clientKey: !ruby/object:Overrides::Terraform::PropertyOverride
```

Reply:
Thanks!

Reply:
> Another build failure, see inline

Thanks @rileykarson. Fixed!

Sorry you have to play message router 🙈